.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: ruby
```

Comment:
There's no tests here. What are you expecting from Travis?

Reply:
Well, the least I wanted is to do some kind of basic sanity checks (e.g. if packages install, if gem builds itself, etc.). If you don't think it should be here, I can remove completely.